File: Selenium/Activities/Activity3.py
# Import required packages from selenium

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver using WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:

    # Launch the URL and get the title
    driver.get("https://v1.training-support.net/selenium/login-form")
    home_title=driver.title
    print ("Title of the Home Page is {}".format(home_title))

    # get the username textbox and type admin
    user_textbox=driver.find_element(By.XPATH, "//input[@id= 'username']")
    user_textbox.send_keys("admin")

    # get the password textbox and type password
    pass_textbox=driver.find_element(By.XPATH, "//input[@name= 'Password']")
    pass_textbox.send_keys("password")

    # get the Submit Button and click on it
    submit_button=driver.find_element(By.XPATH, "//button[text()= 'Log in']")
    submit_button.click()

    # Get the confirmation Text
    confirmation=driver.find_element(By.XPATH, "//div[@id= 'action-confirmation']")
    print("Confirmation Page : " + confirmation.text)

    # CLose the Browser
    driver.close()

File: Selenium/Activities/Activity3.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {
    public static void main(String[] args) {

        // Setup Firefox webdriver
        WebDriverManager.firefoxdriver().setup();

        // Instantiate the webdriver
        WebDriver driver = new FirefoxDriver();

        // Launch the Application URL and print the title of homepage
        driver.get("https://v1.training-support.net/selenium/login-form");
        String homeTitle=driver.getTitle();
        System.out.println("Title of the Home Page is : " + homeTitle);

        // get the username textbox and type admin
        WebElement userTextbox=driver.findElement(By.xpath("//input[@id= 'username']"));
        userTextbox.sendKeys("admin");

        // get the password textbox and type password
        WebElement passTextbox=driver.findElement(By.xpath("//input[@name= 'Password']"));
        passTextbox.sendKeys("password");

        // get the Submit Button and click on it
        WebElement submitButton=driver.findElement(By.xpath("//button[text()= 'Log in']"));
        submitButton.click();

        // Get the confirmation Text
        WebElement confirmation=driver.findElement(By.xpath("//div[@id= 'action-confirmation']"));
        System.out.println("Confirmation Page : " + confirmation.getText());

        // CLose the Browser
        driver.close();
    }
}
